scripts/tasks/gen-project-readme/index.js

```diff
@@ -34,6 +34,7 @@ A modular collection of React hooks.
 🦄 TypeScript support • 🐐 Fully tested • 👾 Server rendering compatible
 
 ![Travis](https://img.shields.io/travis/com/simmo/hooks?style=flat-square)
+![Netlify](https://img.shields.io/netlify/netlify-id?style=flat-square)
```

Comment:
@simmo could you comment here a suggestion with the Netlify project id?

And, if possible, reactivate the Netlify check for the PRs?

Reply:
```suggestion
![Netlify](https://img.shields.io/netlify/ec8abac0-d0d3-4928-8f30-15163c256eb0?style=flat-square)
```

Reply:
Thanks! :smile: